jmix-quartz/quartz-flowui/src/main/java/io/jmix/quartzflowui/QuartzFlowUiConfiguration.java

```diff
@@ -35,6 +36,7 @@
 @ConfigurationPropertiesScan
 @JmixModule(dependsOn = {QuartzConfiguration.class, FlowuiConfiguration.class})
 @PropertySource(name = "io.jmix.quartzflowui", value = "classpath:/io/jmix/quartzflowui/module.properties")
+@Push
```

Comment:
Remove. It shouldn't be here.
It's added automatically to main application class in project.
(Comment was unpublished previous time for some reason)

Reply:
removed